website/sidebars.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
         'expansion',
         'gator',
         'workload-resources',
-        'pubsub',
+        'export',
```

Comment:
How does this impact older versions of docs?

Reply:
It doesn't, with older version of website it will route to `pubsub.md`. And once this PR gets merged, I doubt there would be any maintenance work for old docs since we are changing lots of things in the feature including name.